taqueria-config.ts

```diff
@@ -15,14 +15,14 @@ export type AddTaskCallback = (task: Task.t, plugin: InstalledPlugin.t, handler:
 export const defaultConfig : Config.t = Config.create({
     language: 'en',
     contractsDir: "contracts",
-    testsDir: "tests",
     artifactsDir: "artifacts",
     environment: {
         default: "development",
         development: {
             networks: [],
             sandboxes: ["local"]
-        }
+        },
+        storage: {} as unknown
```

Comment:
Hey @mweichert, I ran into some trouble trying to test a deployment of the increment contract in a test project

When I run `taq deploy`, I get this error:
```
❯ taq originate
No default account has been configured for the sandbox called local. Using the account called bob for origination.
No initial storage provided for test.tz
```

I defined my initial storage as:
```
    "environment": {
        "default": "development",
        "development": {
            "networks": [],
            "sandboxes": [
                "local"
            ]
        },
        "storage": {
            "test.tz": 11
        }
    },
```

Has something changed about the way we need to provide the storage value?

I notice the storage property is now a child of the `environment` property rather than `development` (the defined environment) and thought this may be a cause of the issue

Output with -logPluginRequests:
```
❯ taq originate --logPluginRequests
*** START Call to @taqueria/plugin-taquito ***
node \
/Users/houstonkrohman/code/taqueria/333/node_modules/@taqueria/plugin-taquito/index.js \
--taqRun \
proxy \
--i18n \
'{}' \
--config \
'{"language":"en","contractsDir":"contracts","artifactsDir":"artifacts","network":{},"sandbox":{"local":{"label":"Local Tezos Sandbox","rpcUrl":"http://localhost:20000","protocol":"Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A","accounts":{"bob":{"encryptedKey":"edpkurPsQ8eUApnLUJ9ZPDvu98E8VNj4KtJa1aZr16Cr5ow5VHKnz4","publicKeyHash":"tz1aSkwEot3L2kmUvcoxzjMomb9mvBNuzFK6","secretKey":"unencrypted:edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt"},"alice":{"encryptedKey":"edpkvGfYw3LyB1UcCahKQk4rF2tvbMUk8GFiTuMjL75uGXrpvKXhjn","publicKeyHash":"tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb","secretKey":"unencrypted:edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq"},"john":{"encryptedKey":"edpktt6t2ENhxiQqun6bXPPWC6tFVvNPTDRh1gEPGX4BgDgbDnmGzP","publicKeyHash":"tz1Zwoh1QCVAvJ4sVTojMp9pLYp6Ji4NoZy6","secretKey":"unencrypted:edsk3Un2KRUUoWZufi914HQem96ejVFnkyD8GXRPUSkgqy5etsYXEN"},"jane":{"encryptedKey":"edpkvS6TDSWcqqj3EJi3NRrCMyN7oNw1B3Hp37R19tMThqM8YNhAuS","publicKeyHash":"tz1aHUAC4oviwJuZF1EvVSvFz7cu9KMNYBph","secretKey":"unencrypted:edsk3UkdS1UBfTBkMAoFxmfdmUHmCdNhTzDJ1cSJuZhU1b6k6fZZoQ"},"joe":{"encryptedKey":"edpkuT1QYPYbLLQz9dXhQS33ncsixxeGHbNGmntPTR4VBbWmskHPrV","publicKeyHash":"tz1MVGjgD1YtAPwohsSfk8i3ZiT1yEGM2YXB","secretKey":"unencrypted:edsk3Un2FU9Zeb4KEoATWdpAqcX5JArMUj2ew8S4SuzhPRDmGoqNx2"}}}},"environment":{"default":"development","development":{"networks":[],"sandboxes":["local"]},"storage":{"test.tz":11}},"accounts":{"bob":"3_000_000_000","alice":"3_000_000_000","john":"3_000_000_000","jane":"3_000_000_000","joe":"3_000_000_000"},"plugins":[{"name":"@taqueria/plugin-ligo","type":"npm"},{"name":"@taqueria/plugin-flextesa","type":"npm"},{"name":"@taqueria/plugin-taquito","type":"npm"}],"projectDir":"/Users/houstonkrohman/code/taqueria/333","configFile":"/Users/houstonkrohman/code/taqueria/333/.taq/config.json","hash":"67888303d6a35665106584bc4045b22a65a70f6369f5a8d7c49e263c5a7bc861"}' \
--envVars \
'{}' \
--projectDir \
'./' \
--maxConcurrency \
10 \
--debug \
false \
--logPluginRequests \
true \
--fromVsCode \
false \
--help \
false \
--yes \
false \
--env \
'development' \
--setBuild \
'ba76f04' \
--setVersion \
'dev-623-origination-operation' \
--d \
false \
--p \
'./' \
--y \
false \
--task \
'deploy'
*** END of call to @taqueria/plugin-taquito ***
No default account has been configured for the sandbox called local. Using the account called bob for origination.
No initial storage provided for test.tz
```

Reply:
Ah, thanks Houston. That's a bug. I'll add a fix for that now.

Reply:
Fixed. :)